scale/recipe/models.py

```diff
@@ -110,6 +110,13 @@ def create_recipe(self, recipe_type, data, event, superseded_recipe=None, delta=
         recipe.data = data.get_dict()
         recipe.save()
 
+        # Save models for each recipe input file
+        for input_file_id in data.get_input_file_ids():
+            recipe_file = RecipeFile()
+            recipe_file.recipe_id = recipe.id
+            recipe_file.scale_file_id = input_file_id
+            recipe_file.save()
```

Comment:
Though in practice we don't have a large number of input files for recipes yet, we should go ahead and make this a bulk create. It's possible now to create a recipe with many file inputs and in the future it will be more common.
This would require explicitly setting the created date.